cloudformation/cfncluster.cfn.json

```diff
@@ -3887,6 +3736,48 @@
           "id": "ee5f3006-419e-4786-b527-e9503a662e5e"
         }
       }
+    },
+        "EBSCfnStack": {
+      "Type": "AWS::CloudFormation::Stack",
+      "Properties": {
+        "Parameters":{
+          "AvailabilityZone": {
+            "Ref":"AvailabilityZone"
+          },
+          "VolumeSize": {
+            "Ref" : "VolumeSize"
+          },
+          "VolumeType": {
+            "Ref" : "VolumeType"
+          },
+          "VolumeIOPS": {
+            "Ref" : "VolumeIOPS"
+          },
+          "EBSEncryption":{
+            "Ref" : "EBSEncryption"
+          },
+          "EBSKMSKeyId": {
+            "Ref" : "EBSKMSKeyId"
+          },
+          "EBSVolumeId": {
+            "Ref" : "EBSVolumeId"
+          },
+          "EBSSnapshotId": {
+            "Ref" : "EBSSnapshotId"
+          },
+          "NumberOfEBSVol": {
+            "Ref" : "NumberOfEBSVol"
+          }
+        },
+
+        "TemplateURL":
+          "https://s3.us-east-2.amazonaws.com/shuningc-cfncluster/extra_template/ebs_vol_substack.cfn.json"
```

Comment:
This needs to be dynamically generated and point to non-test bucket :)

Reply:
Since, we are fixing the maximum number of volumes to 5, the template does not need to be dynamically generated. However, I agree that it should point to a bucket which is not dependent on a developer's account :)